pandas/tests/window/moments/test_moments_consistency_rolling.py

```diff
@@ -136,6 +136,54 @@ def test_rolling_apply_consistency(
                 tm.assert_equal(rolling_f_result, rolling_apply_f_result)
 
 
+@pytest.mark.slow
+@pytest.mark.parametrize(
+    "window,min_periods,center", list(_rolling_consistency_cases())
+)
+def test_rolling_groupby(base_functions, window, min_periods, center):
```

Comment:
why are these slow?

Reply:
I was just copying code, removed the slow decorator